action.yaml

```diff
@@ -56,10 +56,18 @@ inputs:
   TF_LOG_PROVIDER:
     description: "Level tf log provider - info, debug, warn or trace"
     required: false
+  CHECKOUT_BRANCH:
+    description: "Whether or not checkout is enabled."
+    required: false
+    default: 'false'
+    
 
 runs:
   using: "composite"
   steps:
+    - name: Checkout
+      if: inputs.CHECKOUT_BRANCH == 'true'
```

Comment:
I would use if `inputs.CHECKOUT_BRANCH != 'false'` instead (if any value other than the default one)